app/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
@dsander weird, it worked on my machine with the file in `app/.bundle`... ah well! Works for me this way too, so I made that change in the patchset I just pushed.